spacy_llm/tasks/templates/ner.v3.jinja

```diff
@@ -0,0 +1,19 @@
+{{ description }}
```

Comment:
We should also support `label_definitions` here similar to the older prompts. That could be a more structured way to provide the description.

Reply:
Yeah I'm wondering what the best way to do this is. The original PromptNER paper has something that resembles label definitions - each entity is kind of fleshed out in the first sentence of the prompt, and the labels are defined in the second sentence, so maybe I could just deterministically generate something similar?

Reply:
I agree, would be great to make this more modular.

Reply:
I would just put the `label_definitions` blurb from the `ner.v2` template right below description. 

Users could provide definitions inline in the description (like how I did it in the example of the PR body) or they could provide the dict of `LABEL: definition` with the description being a simple 1 line description of the task